ragaai_catalyst/tracers/agentic_tracing/tracers/llm_tracer.py

```
@@ -786,12 +858,12 @@ def trace_llm_call_sync(self, original_func, *args, **kwargs):
             raise
 
     def trace_llm(
-        self,
-        name: str = None,
-        tags: List[str] = [],
-        metadata: Dict[str, Any] = {},
-        metrics: List[Dict[str, Any]] = [],
-        feedback: Optional[Any] = None,
+            self,
+            name: str = None,
+            tags: List[str] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)